python/ray/serve/pipeline/tests/test_generate.py

```diff
@@ -0,0 +1,152 @@
+import pytest
+
+import ray
+from ray import serve
+from ray.experimental.dag import InputNode
+from ray.serve.pipeline.generate import (
+    transform_ray_dag_to_serve_dag,
+    extract_deployments_from_serve_dag,
+)
+from ray.serve.pipeline.tests.test_modules import Model, Combine
+
+
+def _validate_consistent_output(
+    deployment, dag, handle_by_name, input=None, output=None
+):
+    """Assert same input lead to same outputs across the following:
+    1) Deployment handle returned from Deployment instance get_handle()
+    2) Original executable Ray DAG
+    3) Deployment handle return from serve public API get_deployment()
+    """
+    deployment_handle = deployment.get_handle()
+    assert ray.get(deployment_handle.remote(input)) == output
+    assert ray.get(dag.execute(input)) == output
+    handle_by_name = serve.get_deployment(handle_by_name).get_handle()
+    assert ray.get(handle_by_name.remote(input)) == output
+
+
+# def test_simple_single_class(serve_instance):
+#     # Assert converting both arg and kwarg
+#     model = Model._bind(2, ratio=0.3)
+#     ray_dag = model.forward._bind(InputNode())
+
+#     serve_root_dag = ray_dag._apply_recursive(
+#         lambda node: transform_ray_dag_to_serve_dag(node)
+#     )
+#     deployments = extract_deployments_from_serve_dag(serve_root_dag)
```

Comment:
why are all of these tests commented out?

Reply:
ah, they passed, but I like to debug with vscode debugger for interactive mode that runs the whole test script and commenting out others save time for each iteration. But I forgot to comment them back on this commit =__=